docs/products/kafka/kafka-connect/reference/s3-sink-additional-parameters-confluent.rst

```diff
@@ -0,0 +1,57 @@
+S3 sink connector by Confluent naming and data formats
+======================================================
+
+The Apache Kafka Connect® S3 sink connector enables you to move data from an Aiven for Apache Kafka® cluster to Amazon S3 for long term storage. The following document describes advanced parameters defining the naming and data formats.
+
+.. Warning::
+
+    Aiven provides, as managed service, two version of S3 sink connector: one developed by Aiven, another developed by Confluent. 
+    
+    This article is about the **Confluent** version. Documentation for the Aiven version is available at the :doc:`dedicated page <s3-sink-additional-parameters>`.
```

Comment:
probably "on the dedicated page"... or "in" :D not sure! 
@TibsAtWork , what is the correct version - "at/on/in the dedicated page"?

Reply:
I put `in`